test/common/README.md

```diff
@@ -60,6 +60,14 @@ On non-Windows platforms, this always returns `true`.
 
 Platform normalizes the `dd` command
 
+### debuglog([fmt, ]msg[, msgs])
+* `fmt` [&lt;string>] Optional format string
+* `msg` [&lt;string>] The string to be printed to stderr
+* `msgs` [&lt;string>] Additional arguments to the function
+
+Prints messages (optionally, formatted using `fmt`) to `process.stderr`,
+using `util.debuglog('test')`.
```

Comment:
Perhaps the wording should clarify that the messages are conditionally printed to stderr based on the value of `NODE_DEBUG `?

Reply:
Or maybe omit the stderr part, e.g.
```markdown
* `msg` [&lt;string>] The string to be printed
```
and 
```markdown
Prints messages (optionally, formatted using `fmt`) using `util.debuglog('test')`.
```